page-faq.php

```diff
@@ -157,11 +159,22 @@
 	<li>The Android mobile app has automatic picture and video upload. Both mobile clients give access to your files and include selective sync for keeping some of your files up to date on the device. Other features are under development or provided by other apps (like calendar and contacts, see <a href="#android">previous FAQ item</a>). If you'd like to add features to the Android or iOS client, find the <a href="https://github.com/owncloud/android/" target="_blank">Android sources here</a> and the <a href="https://github.com/owncloud/ios" target="_blank">iOS sources here</a>. Legal notes: the Android app is under the GPLv2, the iOS app under the GPLv3. To contribute to either, you have to <a href="https://owncloud.org/contribute/agreement/" target="_blank">sign a contributor agreement</a> or contribute your code under the <a href="http://opensource.org/licenses/MIT" target="_blank">MIT license</a>. For testing, we provide a <a href="https://owncloud.org/contribute/iOS-license-exception/" target="_blank">iOS license exception</a> so you can run the iOS app on up to 100 devices.</li>
 </ul>
 
+<a name="calendarcontacts"></a>
+<h3>Why are Calendar and Contacts (or another app) not shipped with the zip file, are they not part of ownCloud?</h3>
+<p>We have four types of apps: Core, Official, Approved and Experimental.Calendar and Contacts are a bit of an issue as, right now, they are not super well maintained and we probably should downgrade them to Approved. If they were core parts of ownCloud users would expect that they get the same attention as ownCloud core. But these apps are developed mostly by students in their free time and thus are not as stable, don’t scale to large instances and don’t get such swift attention to bugreports as the core of ownCloud which is maintained by people who are paid full-time by customers. If a company (or more than one) steps up to pay ownCloud, Inc. for maintenance of the calendar/contact, or if 
```

Comment:
Missing point between: Experimental.Calendar

The sentence also just stops with ", or if " and then the different types of app are listed.